web/templates/topic_edit_form.hbs

```diff
@@ -5,6 +5,5 @@
       autocomplete="off" maxlength="{{ max_topic_length }}" />
     <button type="button" class="topic_edit_save small_square_button animated-purple-button"><i class="fa fa-check" aria-hidden="true"></i></button>
     <button type="button" class="topic_edit_cancel small_square_button small_square_x"><i class="fa fa-remove" aria-hidden="true"></i></button>
-    <div class="alert alert-error edit_error" style="display: none"></div>
```

Comment:
This is the little edit form for changing the topic in a recipient bar -- did you test this? My guess is these code changes won't work for it.

Reply:
Hmm, I don't see `#compose_banners` in this template, so I agree that it may not work. I need to review it more carefully.

Reply:
Does this actually come up? I wouldn't worry about changing it if it almost never happens; there is nothing particularly wrong with "before" (whereas "after" is clearly problematic).